samples/ImageSharp.Web.Sample/Startup.cs

```diff
@@ -37,71 +37,32 @@ public class Startup
         /// </summary>
         /// <param name="services">The collection of service desscriptors.</param>
         public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddImageSharp()
-                .SetRequestParser<QueryCollectionRequestParser>()
-                .Configure<PhysicalFileSystemCacheOptions>(options =>
-                {
-                    options.CacheRootPath = null;
-                    options.CacheFolder = "is-cache";
-                    options.CacheFolderDepth = 8;
-                })
-                .SetCache<PhysicalFileSystemCache>()
-                .SetCacheKey<UriRelativeLowerInvariantCacheKey>()
-                .SetCacheHash<SHA256CacheHash>()
-                .Configure<PhysicalFileSystemProviderOptions>(options =>
-                {
-                    options.ProviderRootPath = null;
-                })
-                .AddProvider<PhysicalFileSystemProvider>()
-                .AddProcessor<ResizeWebProcessor>()
-                .AddProcessor<FormatWebProcessor>()
-                .AddProcessor<BackgroundColorWebProcessor>()
-                .AddProcessor<QualityWebProcessor>();
-
-            // Add the default service and options.
-            //
-            // services.AddImageSharp();
-
-            // Or add the default service and custom options.
-            //
-            // this.ConfigureDefaultServicesAndCustomOptions(services);
-
-            // Or we can fine-grain control adding the default options and configure all other services.
-            //
-            // this.ConfigureCustomServicesAndDefaultOptions(services);
-
-            // Or we can fine-grain control adding custom options and configure all other services
-            // There are also factory methods for each builder that will allow building from configuration files.
-            //
-            // this.ConfigureCustomServicesAndCustomOptions(services);
-        }
+            => services.AddImageSharp(); // Add the default service and options
```

Comment:
The previous code duplicated the defaults that were already set in `AddImageSharp()` and is already documented on https://docs.sixlabors.com/articles/imagesharp.web/gettingstarted.html. Maybe the examples below can also be cleaned up?

Reply:
They’re for example purpose only. People don’t always read docs.

Reply:
OK, I'll leave the examples in the `ConfigureDefaultServicesAndCustomOptions`, `ConfigureCustomServicesAndDefaultOptions` and `ConfigureCustomServicesAndCustomOptions` methods 👍🏻